README.md

```diff
@@ -1,8 +1,9 @@
 # B01LER
-![B01LER_UpdateImg_1100-480@2x](https://github.com/deven-org/boiler/assets/122102805/7f55e1fd-955f-4c9f-890d-3d5a03f5c3aa)
 
+![B01LER_UpdateImg_1100-480@2x](https://github.com/deven-org/boiler/assets/122102805/7f55e1fd-955f-4c9f-890d-3d5a03f5c3aa)
 
 ## :page_with_curl: Content
+
```

Comment:
.md file does not have the new line after the heading, please for consistency purporse, remove the newly added lines throughout the file. New line might have been added by the vscode, prettier-code formatter extension.